### ISSUE TITLE
Cancelamento de compra pelos caixas

### ISSUE THREAD

**Issue by fabianogcarvalho** [fabianogcarvalho/portasabertas_web#10]
A opção de cancelamento da compra pelo caixa existe mas na época pediram pra desativar

**Comment by fabianogcarvalho** [comment]
Eu implementei a action na tela dos caixas para este cancelamento. Precisamos decidir se os caixas podem cancelar vendas ou não

**Comment by otaviogrrd** [comment]
Concordo que os caixas tenham acesso à essa funcionalidade. Ainda mais com os caixas itinerantes, eles podem fazer isso mais próximo do cliente
